src/Middleware/StaticFiles/src/StaticFileContext.cs

```diff
@@ -388,8 +388,8 @@ internal async Task SendRangeAsync()
         // Note: This assumes ranges have been normalized to absolute byte offsets.
         private ContentRangeHeaderValue ComputeContentRange(RangeItemHeaderValue range, out long start, out long length)
         {
-            start = range.From.Value;
-            long end = range.To.Value;
+            start = range.From ?? 0;
+            long end = range.To ?? 0;
```

Comment:
This is misleading, From and To's values should always have been set by now or the code wouldn't work at all.